examples/cluster/plot_cluster_comparison.py

```diff
@@ -79,6 +79,9 @@
     "min_samples": 7,
     "xi": 0.05,
     "min_cluster_size": 0.1,
+    "allow_single_cluster": True,
+    "hdbscan_min_cluster_size": 15,
+    "hdbscan_min_samples": 3,
```

Comment:
Using separate keys would make the dataset-specific parameter updates a bit more verbose and potentially decrease clarity, so perhaps enforcing namespace via the string itself (e.g. `hdbscan_min_samples`) for the other params where possible would be safer.

With that being said, I suspect it would be better to relegate that to a separate PR to keep this one ~~simple~~ somewhat manageable :)

Reply:
Could you please then open an issue for this so that we don't forget?